docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+#Import Node base image
+FROM node:12-slim
+
+#Install Git
+RUN apt-get update && \
+    apt list -a git && \
+    apt-get install -y --no-install-recommends git=1:2.*
+
+#Clone SDK repo to app directory
+RUN git clone https://github.com/watson-developer-cloud/node-sdk.git
+
+#Install the SDK
+RUN npm install ibm-watson
+
+#Set working directory to node-sdk
+WORKDIR /node-sdk
+
+#Install dependencies and build to run tests.
+RUN npm install && \
+    npm run build
+
+#Run tests to verify installation
+CMD ["npm" , "test"]
```

Comment:
This is just a suggestion - `npm test` runs the full suite of integration tests, which have a variable (and unfortunately high) failure rate due to service timeouts, etc. They also take ~25 minutes to run on average. It may be better to run only the unit tests:

`CMD ["npm" , "run" , "test-unit"]`